lte/gateway/c/core/oai/tasks/nas/esm/esm_ebr.c

```diff
@@ -261,7 +261,9 @@ status_code_e esm_ebr_start_timer(
   ue_mm_context_t* ue_mm_context =
       PARENT_STRUCT(emm_context, struct ue_mm_context_s, emm_context);
   if (ue_mm_context == NULL) {
-    OAILOG_ERROR(LOG_NAS_ESM, "ESM-FSM   - ue mme context null..\n");
+    OAILOG_ERROR_UE(
```

Comment:
ue_mm_context does not exist in this case, let's not change this

Reply:
Resolved